pkg/console/operator/sync_v400.go

```diff
@@ -501,3 +509,38 @@ func getConsoleURL(route *routev1.Route) (string, error) {
 	}
 	return util.HTTPS(host), nil
 }
+
+// return plugins that are:
+// - enabled in the console-operator config
+// - created and available as a CR on the cluster
+func getEnabledPlugins(operatorConfig *operatorv1.Console, pluginsList []v1.ConsolePlugin) []string {
+	availablePluginsNames := []string{}
```

Comment:
use the power of sets instead of having to write this on your own https://github.com/kubernetes/kubernetes/blob/7d72ddca6e9a4e9cfa481b137bd0546be6a16cad/vendor/k8s.io/apimachinery/pkg/util/sets/string.go#L30